### PR TITLE
Added more content to CONTRIBUTING.md

### LINKED ISSUE #16755: Improvements for CONTRIBUTING.md
This issue is created to discuss the changes that can be made to [CONTRIBUTING.md](https://github.com/sympy/sympy/blob/master/CONTRIBUTING.md) and [introduction to contributing](https://github.com/sympy/sympy/wiki/Introduction-to-contributing) wiki to make them more beginner friendly.

Any suggestions regarding how to improve the wiki and CONTRIBUTING.md is appreciated.

For more information, refer to #16742

### PR DESCRIPTION
**Pull request by czgdp1807**
#### References to other Issues or PRs

Fixes https://github.com/sympy/sympy/issues/16755

#### Brief description of what is fixed or changed
N/A

#### Other comments
N/A

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by czgdp1807** [comment]
@asmeurer Please take a look and let me know of any changes to be made.

**Comment by Sc0rpi0n101** [comment]
Can you do something similar for pull requests too and maybe link [this](https://github.com/sympy/sympy/wiki/Development-workflow#create-a-patch-file-or-pull-request-for-github) for more info?

Is there a similar section for Issues in the wiki pages?

**Comment by czgdp1807** [comment]
@Sc0rpi0n101 There is already a template for pull requests so adding to CONTRIBUTING.md would be redundant. Adding on wiki is also redundant according to me as people will go to CONTRIBUTING.md first while creating an issue, so, writing the same/similar thing on wiki will not be useful.

**Comment by oscarbenjamin** [comment]
I'm not sure how useful this additional text is. I think if we had a page on the wiki that had non-obvious tips as well as general instruction with meaningful examples then that would be useful to link to from other places.

Another point in regards to opening an issue is that I think a lot of issues would be better started as mailing list discussions. Only when it is established that there is possibly something to improve in SymPy should an issue be opened (Github issues should not be for getting help with using SymPy).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ looking for some way to contribute a good place to start is to look at the
 issues tagged [Easy to Fix](
 https://github.com/sympy/sympy/issues?q=is%3Aopen+is%3Aissue+label%3A%22Easy+to+Fix%22).
 
+If you are raising an issue for the first time on our issue tracker then, please
+take a look at the following points:
+
+1. Please give a short descriptive title relevant to your issue.
+2. If you are facing problem with a particular functionality of sympy
```
Comment (smichr) [inline CONTRIBUTING.md]:
```suggestion
2. If you are facing problem with a particular functionality of SymPy
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ looking for some way to contribute a good place to start is to look at the
...
+   then please include the code that reproduces the probelem with the
```
Comment (smichr) [inline CONTRIBUTING.md]: probelem -> problem

**Comment by smichr** [comment]
Since you reacted favorably to a comment that questioned how useful this is I am not sure if you still want to include this...or if a comment about raising a question on mailing list would be good to add.

**Comment by czgdp1807** [comment]
@smichr I think this can be safely closed. And @Sc0rpi0n101 can edit the wiki for enhancing the workflow. That's all from my side.
PS - Apologies for the late  reply.

**Comment by Sc0rpi0n101** [comment]
>  And @Sc0rpi0n101 can edit the wiki for enhancing the workflow. 
Sure, I can do that.

**Pull request closed without merging**